### PR TITLE
TST: Test Case Addition + Scrapy Fixes as needed + Browser Actions(minor fix)

### PR DESCRIPTION
**Pull request by gnana03**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: Examples/sampleseleniumtest.py [hunk 1]
```diff
@@ -1,42 +1,195 @@
...
     """hold all locators for POM style implementation."""
 
-    base_url = "http://www.python.org"
-    two_url = "https://xkcd.com/979/"
-    search_box = {"locatorvalue": 'id-search-field',
-                  "by": "By.ID", "value": "selenium " + Keys.ENTER}
-    button = {"locatorvalue": '#topLeft > ul > li:nth-child(1) > a',
-              "by": "By.CSS_SELECTOR"}
+    def __init__(self):
+        print("This class contains all pageobjects")
```
Comment (surevs) [inline Examples/sampleseleniumtest.py]: Please remove print statement

File: Examples/sampleseleniumtest.py [hunk 2]
```diff
@@ -1,42 +1,195 @@
...
+        self.go_back()
+        self.page_readiness_wait()
+        time.sleep(5)
+        self.click(PageObjects.links_all.get('dynamic_controls'))
+        self.click(PageObjects.buttons_all.get('enable_btn'))
+        time.sleep(5)
+        self.send_keys(PageObjects.text_boxes.get('text_box'))
+        time.sleep(5)
+        self.switch_to_default_content()
+        path = os.getcwd()
```
Comment (surevs) [inline Examples/sampleseleniumtest.py]: can be merged and re written as self.capture_screenshot(os.getcwd()+"\\example_Screenshot1.png")

File: imgqa/setup_scrapper.tmpl [hunk 3]
```diff
@@ -14,29 +13,27 @@ class Spider(scrapy.Spider):
...
+    allowed_domains.append(urlparse(url).netloc)
     rules = (Rule(SgmlLinkExtractor(), callback='parse', follow=False),)
     list2 = []
     list3=[]
     def parse(self, response):
         list1=[str(res.extract()) for res in response.xpath('//a/@href')
                 if str(res.extract()).startswith('http:')]
         for url in response.xpath('//a/@href'):
-            QuotesSpider.list2.append(
+            Spider.list2.append(
```
Comment (surevs) [inline imgqa/setup_scrapper.tmpl]: Please avoid camel casing

Comment (obannak) [inline imgqa/setup_scrapper.tmpl]: please avoid using camelcase letters. use spider instead of Spider

File: imgqa/setup_scrapper.tmpl [hunk 4]
```diff
@@ -14,29 +13,27 @@ class Spider(scrapy.Spider):
...
                           str(url.extract()))
             yield url.extract()
-        for URLtoHit in set(QuotesSpider.list2):
+        for URLtoHit in set(Spider.list2):
```
Comment (obannak) [inline imgqa/setup_scrapper.tmpl]: please avoid using camelcase letters.

File: imgqa/setup_scrapper.tmpl [hunk 5]
```diff
@@ -14,29 +13,27 @@ class Spider(scrapy.Spider):
...
             next_link_to_hit=response.urljoin(URLtoHit)
-            if(next_link_to_hit not in set(QuotesSpider.list3)):
+            if(next_link_to_hit not in set(Spider.list3)):
```
Comment (obannak) [inline imgqa/setup_scrapper.tmpl]: please avoid using camelcase letters.

File: imgqa/setup_scrapper.tmpl [hunk 6]
```diff
@@ -14,29 +13,27 @@ class Spider(scrapy.Spider):
...
-
-        path = os.getcwd()
-        with open(path + "\\crawler.csv", "wb") as f:
+            Spider.list3.append(next_link_to_hit)
+        with open("Result.csv", "wb") as f:
             writer = csv.writer(f)
-            writer.writerows(set(QuotesSpider.list2))
-        outF = open(path + "\\crawler.txt", "w")
-        for line in set(QuotesSpider.list2):
+            writer.writerows(set(Spider.list2))
```
Comment (obannak) [inline imgqa/setup_scrapper.tmpl]: please avoid using camelcase letters.

**Pull request merged**